OpenSim/Common/C3DFileAdapter.h

```diff
@@ -45,14 +80,27 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
     ~C3DFileAdapter()                                = default;
 
     C3DFileAdapter* clone() const override;
+
+    void setLocationForForceExpression(const ForceLocation location) {
+        _location = location;
+    }
+
+    const ForceLocation getLocationForForceExpression() const {
+        return _location;
+    }
     
     /** Read in a C3D file into separate markers and forces tables of type
         TimeSeriesTableVec3. The markers table has each column labeled by its
         corresponding marker name. For the forces table, the data are grouped
         by sensor (force-plate #) in force, point and moment order, with the
-        respective *f#*, *p#* and *m#* column labels. */
+        respective *f#*, *p#* and *m#* column labels. C3DFileAdpater provides
+        options for expressing the force-plate measurements either as the
+        net force and moments expressed at the ForcePlateOrigin, the 
+        CentereOfPressure, or the PointOfWrenceApplication (PWA) as defined by
+        Shimba 1984*/
```

Comment:
Has the complete reference to "Shimba 1984" been provided in the doxygen somewhere? I can't find it in this file. You probably mean "An estimation of center of gravity from force platform data" which is the first hit on Google Scholar, but it might be good to be explicit here since many users may be unfamiliar with wrenches. :wrench: